app/controllers/finance/bank_accounts_controller.rb

```diff
@@ -5,6 +5,14 @@ def index
     redirect_to finance_bank_account_transactions_url(@bank_accounts.first) if @bank_accounts.count == 1
   end
 
+  def assign_unchecked_transactions
+    @bank_account = BankAccount.find(params[:id])
+    count = @bank_account.assign_unchecked_transactions
+    redirect_to finance_bank_account_transactions_url(@bank_account), notice: t('finance.bank_accounts.controller.import.notice', :count => count)
```

Comment:
I would somehow expect to be able to review all transactions. If I were a foodcoop beginning to use this feature, I would feel uneasy if the system does something I may not fully comprehend yet. Seeing what it does, and checking it, would make me feel a lot better about it.